docs/csharp/programming-guide/unsafe-code-pointers/how-to-obtain-the-address-of-a-variable.md

```diff
@@ -29,7 +29,7 @@ int* p = &number; //address-of operator &
  You cannot get the address of a constant or a value.  
   
 ## Example  
- In this example, a pointer to `int`, `p`, is declared and assigned the address of an integer variable, `number`. The variable `number` is initialized as a result of the assignment to *p. If you make this assignment statement a comment, the initialization of the variable `number` will be removed, but no compile-time error is issued. Notice the use of the [Member Access](../../../csharp/programming-guide/unsafe-code-pointers/how-to-access-a-member-with-a-pointer.md) operator `->` to obtain and display the address stored in the pointer.  
+ In this example, a pointer to `int`, `p`, is declared and assigned the address of an integer variable, `number`. The variable `number` is initialized as a result of the assignment to *p. If you make this assignment statement a comment, the initialization of the variable `number` will be removed, but no compile-time error is issued.  
   
  [!code-csharp[csProgGuidePointers#7](../../../csharp/programming-guide/unsafe-code-pointers/codesnippet/CSharp/how-to-obtain-the-address-of-a-variable_1.cs)]  
```

Comment:
A separate code block with a single-line comment is awkward. Could you delete line 34 and change line 36 to \[!code-csharp\[pointers\#8\](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuidePointers/CS/Pointers.cs#8)]? (I think that the link is correct.) Rather than adding the include comment to the source, could you also add a note between the existing text and the example:

\> \[!NOTE]
\>  Complile this example with the \`-unsafe\` compiler option.